crates/router/src/connector/aci.rs

```diff
@@ -102,16 +107,25 @@ impl
         &self,
         req: &types::PaymentsSyncRouterData,
         _connectors: &settings::Connectors,
-    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
-        let mut header = vec![
+    ) -> CustomResult<Vec<(String, Box<dyn services::request::HeaderValue>)>, errors::ConnectorError>
```

Comment:
Consider using a type alias for `Box<dyn services::request::HeaderValue>` or `Vec<(String, Box<dyn services::request::HeaderValue>)>` maybe?